packages/axelar-wasm-std/src/response.rs

```diff
@@ -13,9 +13,9 @@ pub enum Error {
 
 /// Get a msg wrapped inside a `WasmMsg::Execute` from a `Response`.
 /// If there are no wasm messages or more than one message in the response, this returns an error.
-pub fn inspect_response_msg<T>(response: Response) -> Result<T, Error>
+pub fn inspect_response_msg<T, D>(response: Response<T>) -> Result<D, Error>
 where
-    T: DeserializeOwned,
+    D: DeserializeOwned,
```

Comment:
Use `Empty` as the default similar to what `Response` does?
```suggestion
pub fn inspect_response_msg<T, D = Empty>(response: Response<D>) -> Result<T, Error>
where
    T: DeserializeOwned,
```

Reply:
na, default is not allowed for function definations